packages/l/libsdl_image/xmake.lua

```diff
@@ -43,19 +52,16 @@ package("libsdl_image")
 
     on_install("macosx", "linux", function (package)
         local configs = {}
-        if package:config("shared") then
-            table.insert(configs, "--enable-shared=yes")
-        else
-            table.insert(configs, "--enable-shared=no")
-        end
-        if package:is_plat("linux") and package:config("pic") ~= false then
-            table.insert(configs, "--with-pic")
-        end
+        table.insert(configs, "--enable-shared=" .. (package:config("shared") and "yes" or "no"))
+        table.insert(configs, "--enable-static=" .. (package:config("shared") and "no" or "yes"))
         local libsdl = package:dep("libsdl")
         if libsdl and not libsdl:is_system() then
             table.insert(configs, "--with-sdl-prefix=" .. libsdl:installdir())
         end
-        io.replace("Makefile.am", "noinst_PROGRAMS = showimage", "")
+        if libsdl and package:is_plat("macosx") then
+            table.insert(configs, "-DSDL2_DIR=" .. libsdl:installdir())
```

Comment:
configure: error: unrecognized option: `-DSDL2_DIR=/Users/runner/.xmake/packages/l/libsdl/2.24.0/452ad75fc0274e7eb90336efbef13ba2'
Try `./configure --help' for more information